index.js

```diff
@@ -39,7 +39,13 @@ module.exports = (app) => {
       app.log(`PR #${pr.number}: Request received`);
     }
 
-    let outstandingTasks = checkOutstandingTasks(pr.body);
+    let outstandingTasks = checkOutstandingTasks(null);
+
+    // if the author is a renovate bot, ignore checks
+    // https://www.mend.io/free-developer-tools/renovate/
+    if (pr.user.login.indexOf('renovate[bot]') === -1) {
+      outstandingTasks = checkOutstandingTasks(pr.body);
+    }
```

Comment:
Looks good but could we flip this statement around so when reading this and adding other bots to be excluded in the future, we default to having the body set but then unset it on exception like so:

```suggestion
    let prBody = pr.body;
    
    // if the author is a renovate bot, ignore checks
    // https://www.mend.io/free-developer-tools/renovate/
    if (pr.user.login.indexOf('renovate[bot]') !== -1) {
      prBody = null;
    }
    
    let outstandingTasks = checkOutstandingTasks(prBody);

```

Reply:
Great suggestion, thank you @stilliard